editor/components/block-list/style.scss

```diff
@@ -581,17 +581,27 @@
 		}
 	}
 
-	// Show side UI inline below the block on mobile
+	// Show side UI inline below the block on mobile.
 	.editor-block-list__block-mobile-toolbar {
 		display: flex;
 		flex-direction: row;
-		margin-top: $item-spacing;
+		margin-top: $item-spacing + $block-toolbar-height; // Make room for the formatting bar above. 
+		margin-right: -$block-padding;
+		margin-bottom: -$block-padding - 1px; // Include border.
+		margin-left: -$block-padding;
+		border-top: 1px solid $light-gray-500;
```

Comment:
Yeah, looks like this `1px` could be a variable, that'd help with the `// include border` comment above.